vlib/v/fmt/tests/empty_lines_expected.vv

```diff
@@ -0,0 +1,56 @@
+fn keep_single_empty_line() {
+	println('a')
+	println('b')
+
+	println('c')
+
+	d := 0
+
+	if true {
+		println('e')
+	}
+
+	f := 0
+}
```

Comment:
should be moved to empty_lines_keep.vv

Reply:
Yes indeed, `_keep.vv` files are easier to maintain, since there is a single file.
The _input/_expected pair should be used only in rare cases, where you need to test that vfmt can do cleanup, not just keeping formatting file the same.
For example, a small _input/_expected pair can test that multiple empty lines are collapsed in just one empty line.
Everything else can be checked in a `_keep.vv` file.

Reply:
fixed